src/ORM/Behavior/CounterCacheBehavior.php

```diff
@@ -79,6 +81,51 @@ class CounterCacheBehavior extends Behavior
 {
 
     /**
+     * Store the fields which should be ignored
+     *
+     * @var array
+     */
+    protected $_ignoreIfDirty = [];
+
+    /**
+     * beforeSave callback.
+     *
+     * Check if a field, which should be ignored, is dirty
+     *
+     * @param \Cake\Event\Event $event The beforeSave event that was fired
+     * @param \Cake\Datasource\EntityInterface $entity The entity that is going to be saved
+     * @return void
+     */
+    public function beforeSave(Event $event, EntityInterface $entity)
+    {
+        foreach ($this->_config as $assoc => $settings) {
+            $assoc = $this->_table->association($assoc);
+            foreach ($settings as $field => $config) {
+                if (is_int($field)) {
+                    $field = $config;
+                    $config = [];
+                }
+
+                $regitsryAlias = $assoc->target()->registryAlias();
+                $entityAlias = strtolower(Inflector::singularize($regitsryAlias));
+
+                if (!is_callable($config) &&
+                    isset($config['ignore_if_dirty']) &&
```

Comment:
Mind you casing. Check out what markstory's proposal actually was: ignoreIfDirty.